.cirrus.yml

```diff
@@ -28,7 +28,7 @@ test_task:
     - poetry config virtualenvs.in-project true
   test_script:
     - poetry install
-    - poetry run pytest -q --junitxml=junit.xml tests
+    - poetry run python -m pytest -vv -p no:clarity -p no:sugar --junitxml=junit.xml tests/
```

Comment:
Is there a reason why we do this? ie. are we expection pytest  from outside the env to be used somehow?

Reply:
No -- it's done for consistency with the Github action.